x-pack/plugins/canvas/canvas_plugin_src/functions/common/render.ts

```diff
@@ -26,25 +26,43 @@ export function render(): ContextFunction<'render', Render<any>, Arguments, Rend
     },
     args: {
       as: {
-        types: ['string', 'null'],
+        types: ['string'],
         help: argHelp.as,
-        options: ['debug', 'error', 'image', 'pie', 'plot', 'shape', 'table', 'text'],
+        options: [
+          'advanced_filter',
+          'debug',
+          'dropdown_filter',
+          'error',
+          'image',
+          'markdown',
+          'metric',
+          'pie',
+          'plot',
+          'progress',
+          'repeatImage',
+          'revealImage',
+          'shape',
+          'table',
+          'time_filter',
+          'text',
+        ],
       },
       css: {
-        types: ['string', 'null'],
-        default: '"* > * {}"',
+        types: ['string'],
         help: argHelp.css,
+        default: `"${DEFAULT_ELEMENT_CSS}"`,
       },
       containerStyle: {
-        types: ['containerStyle', 'null'],
+        types: ['containerStyle'],
         help: argHelp.containerStyle,
+        default: '{containerStyle}',
       },
     },
     fn: (context, args) => {
       return {
         ...context,
         as: args.as || context.as,
-        css: args.css,
+        css: args.css || DEFAULT_ELEMENT_CSS,
```

Comment:
q: Is this required? The argument has a default value... are you just guarding against that value ending up being `null`?

Reply:
Yep, guarding against `null` as well as `""`. `style-it` doesn't play nice when it gets an empty string or `null` as the style. 
![Screen Shot 2019-06-05 at 2 51 08 PM](https://user-images.githubusercontent.com/1697105/58993169-932a0600-87a1-11e9-9b20-e9232924a0c9.png)